### DISCUSSION TITLE
Lay-z-spa Helsinki 2019 working :-)

### DISCUSSION THREAD

**Discussion by neildotwilliams** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#115] [Show and tell]
Hi. Just confirming this works on my Helsinki model with pump version #54189. The main controller board had 2019 on it. I had to replace the spindle in the pump so I added this fantastic WiFi upgrade while I had it apart. Works great with Home Assistant (MQTT, node red, influxdb, grafana). Loving my Lay-Z-WiFi!

![image](https://user-images.githubusercontent.com/8913897/132134701-d88b9ea5-e75e-4ae7-b1d5-d088988b4cb2.jpeg)
![image](https://user-images.githubusercontent.com/8913897/132134712-6613f26b-3002-476d-9c3d-5ceb0907be53.jpeg)
![image](https://user-images.githubusercontent.com/8913897/132134725-77ea2737-17e4-4940-9503-b7444bc6e164.jpeg)

**Comment by visualapproach** [comment]
Awesome! It makes me happy to hear when other people like it and succeed with the build. Thank you for posting!
/Visualapproach
